core/ui/settings_general.cpp

```diff
@@ -400,7 +387,19 @@ void gui_settings_general()
         "Folders containing textures/<gameId> or <gameId> under a textures subfolder");
     ImGui::Spacing();
 
-#if !defined(__ANDROID__)
+    // New custom path lists
+    managePathList("Box Art Folders", config::BoxartPath.get(),
+        "Folders containing box art images (png/jpg). If empty, Flycast will use the default Home Folder/boxart for downloads and generated art");
+    ImGui::Spacing();
```

Comment:
I find it confusing for users to show a path list when only the first one will be used. I changed for the texture dump path and it would make sense to do the same for the boxart folder.